go.mod

```diff
@@ -4,7 +4,7 @@ go 1.21.1
 
 require (
 	github.com/bufbuild/buf v1.29.0
-	github.com/conduitio/conduit-commons v0.0.0-20240103200651-5a5746611a8e
+	github.com/conduitio/conduit-commons v0.0.1
```

Comment:
@hariso should this be https://github.com/ConduitIO/conduit-commons/releases/tag/v0.1.1 instead?

Reply:
Yup!